### PR TITLE
Fix tests for PUT endpoint

### PR DESCRIPTION
**Pull request by jirkasemmler**
Jira: [CT-1270](https://keboola.atlassian.net/browse/CT-1270)
Connection PR: https://github.com/keboola/connection/pull/4883/files
SAPI PR:  https://github.com/keboola/storage-api-php-client/pull/1283

storage backend command: https://github.com/keboola/storage-backend/pull/132
storage backend table utils: https://github.com/keboola/storage-backend/pull/129
BQ driver: https://github.com/keboola/php-storage-driver-bigquery/pull/117
Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] zdokumentovat chovani BQ
- [x] edge case testy - failujici scenaria

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

[CT-1270]: https://keboola.atlassian.net/browse/CT-1270?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -2465,23 +2465,24 @@ If there is no column datatype specified (column.definition.type or column.baset
 
 ### Update Column Definition [PUT /v2/storage/tables/{table_id}/columns/{column_name}/definition]
 
-Updates a column of a table with typed columns. *Snowflake only.*
+Updates a column of a table with typed columns. *Snowflake, BigQuery only.*
```
Comment (jirkasemmler) [inline apiary.apib:2468]: TODO lepe zdokumentovat jak se vrati chyba

**Review by romanbracinik — APPROVED** [review]
🙆

**Comment by romanbracinik** [comment]
rebase
